### PR TITLE
Image & input components

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
image

Responsive and round Images
image

### CODE REVIEW

File: pages/components/input.html [hunk 1]
```diff
@@ -106,8 +106,60 @@ <h1 clss="page-title">input</h1>
...
+        </div>
+        <div class="form-item status-warning-outline">
+          <input type="password" class="form-input" placeholder="password" aria-label="password">
+        </div>
+        <div class="form-item status-error-outline">
+          <input type="password" class="form-input" placeholder="confirm password" aria-label="confirm password">
+        </div>
+            
+        <fieldset class="form-item status-plain-outline">
+          <legend>Gender</legend>
```
Comment (manujagobind) [inline pages/components/input.html]: I didn't knew about fieldset and legend :)

File: pages/components/input.html [hunk 2]
```diff
@@ -106,8 +106,60 @@ <h1 clss="page-title">input</h1>
...
+            <p>
+              <label for="JavaScript">JavaScript</label>
+              <input type="checkbox" name="gender" id="JavaScript" checked class="">
+            </p>
+            <p>
+              <label for="Python"> Python </label>
+              <input type="checkbox" name="gender" id="Python" class="">
+            </p>
+            <p>
+              <label for="Java"> Java </label>
```
Comment (manujagobind) [inline pages/components/input.html]: Can remove spaces after > and before <

File: pages/components/input.html [hunk 3]
```diff
@@ -106,8 +106,60 @@ <h1 clss="page-title">input</h1>
...
+              <input type= "checkbox" name="gender" id="Java" class="">
+            </p>
+          </fieldset>
+
+          <div class="form-item status-plain-outline">
+            <label for="codingLevel">Coding Level:</label>
```
Comment (manujagobind) [inline pages/components/input.html]: Use similar case for class name. coding-level would have been more consistent.

File: styles/05-objects/input.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
+
+.form{
```
Comment (manujagobind) [inline styles/05-objects/input.css]: Add space before {

Remove blank first line

File: pages/components/input.html [hunk 5]
```diff
@@ -106,8 +106,60 @@ <h1 clss="page-title">input</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <form action="#" class="form">
+        <div class="form-item status-success-outline">
```
Comment (manujagobind) [inline pages/components/input.html]: I think "outline" is redundant in the class name for status if we don't have any other type of status (besides outline)

File: styles/05-objects/input.css [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+.form-item input[type=range]::before,
+.form-item input[type=range]::after {
+  display: inline-block;
+  position: absolute;
+  color: var(--color-status-plain);
+}
+
+.form-item input[type=range]::before {
+  content: attr(min);
```
Comment (manujagobind) [inline styles/05-objects/input.css:112]: Didn't know we could do this. :)

**Review by manujagobind — APPROVED** [review]
Excellent. I think I need to go back and improve my PR after learning new things from here.

File: pages/components/image.html [hunk 7]
```diff
@@ -114,10 +114,25 @@ <h1 clss="page-title">image</h1>
...
+    
+    <div>
+      <img src="https://picsum.photos/100" 
+      alt="logo"
+      srcset="
+           https://picsum.photos/200 200w,
+           https://picsum.photos/400 400w,
+           https://picsum.photos/600 600w,
+           
+       "
```
Comment (HrishikeshMahalle) [inline pages/components/image.html]: good use of Srcset!!

**Review by HrishikeshMahalle — APPROVED** [review]

File: pages/components/input.html (hunk 5 above)
Comment (HarshnaNHaswani) [reply pages/components/input.html]: My status classes are background and outline and I have reused these classes everywhere like alert, badge etc. so I need to differentiate them

**Comment by HarshnaNHaswani** [comment]
Thanks for your time and help @HrishikeshMahalle @manujagobind. Will work on the mentioned changes

**Pull request merged**